tests/bsim/bluetooth/ll/bis/src/main.c

```diff
@@ -784,6 +873,14 @@ static void test_iso_recv_main(void)
 	printk("success.\n");
 #endif
 
+	for (uint8_t chan = 0U; chan < CONFIG_BT_ISO_MAX_CHAN; chan++) {
```

Comment:
```suggestion
	for (int chan = 0U; chan < CONFIG_BT_ISO_MAX_CHAN; chan++) {
```

CONFIG int values are signed integers